.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,24 @@
+name: GitHub Actions Demo
```

Comment:
It would be nice to rename this file, workflow and job name to something similar to "Run clarity-ext unit tests"

Reply:
Yes, when I arrived at home yesterday, I realized that I forgot to rename the demo file :)